plugins/org.jkiss.dbeaver.ext.postgresql/src/org/jkiss/dbeaver/ext/postgresql/edit/PostgreTableColumnManager.java

```diff
@@ -146,14 +149,20 @@ public DBSObjectCache<? extends DBSObject, PostgreTableColumn> getObjectsCache(P
 
     protected ColumnModifier[] getSupportedModifiers(PostgreTableColumn column, Map<String, Object> options)
     {
+        PostgreServerExtension serverType = column.getDataSource().getServerType();
+        if (serverType.getServerTypeName().equals(POSTGRE_SERVER_TYPE_CRATE_DB)) {
```

Comment:
Let's add function to ServerType (e.g. supportsNotNull)

Reply:
Done